electron-utils/oauth.ts

```diff
@@ -81,10 +86,12 @@ function getAuthorizationCode(parentWindow: BrowserWindow, repoPermissionLevel:
       const query = url_parts.query;
       const code = query.code;
       const error = query.error;
-      const state = query.state;
+      const returnedState = query.state;
 
-      if (error !== undefined && state !== undefined) {
+      if (error !== undefined) {
         reject(error);
+      } else if (!isReturnedStateSame(state, returnedState)) {
+        reject(new Error(stateErrorMessage));
```

Comment:
Same here; we can just add a check to `!isReturnedStateSame` before resolving the code. No need to throw the error.